cmd/configurator/main.go

```diff
@@ -80,7 +75,9 @@ func main() {
 	log.Println("Configurator is finished")
 }
 
-func watchPartitions(ctx context.Context, wg *sync.WaitGroup, slurmClient api.WorkloadManagerClient, k8sClient *corev1.CoreV1Client) {
+func watchPartitions(ctx context.Context, wg *sync.WaitGroup,
+	slurm api.WorkloadManagerClient, k8s *corev1.CoreV1Client) {
```

Comment:
those names give me a feeling that we are using Pkg functions instead client methods

Reply:
very confusing to read it